### DISCUSSION TITLE
Welcome to tree-sitter Discussions!

### DISCUSSION THREAD

**Discussion by dcreager** [tree-sitter/tree-sitter#751] [General]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vigoux** [comment]
Hi everyone, I am part of [nvim-treesitter](https://github.com/nvim-treesitter/), and org that works on integrating tree-sitter and neovim !

We trully think that tree-sitter is the way to enhance many features of neovim (and vim) !

Thank you all for your work, and I'll let the other member of the team present themselves !

**Comment by razzeee** [comment]
Hey beings,

I'm mostly working on tree-sitter-elm https://github.com/Razzeee/tree-sitter-elm/ as I'm using it to feed our language server at https://github.com/elm-tooling/elm-language-server

**Comment by sogaiu** [comment]
Hi all,

I'm working on [tree-sitter-clojure](https://github.com/sogaiu/tree-sitter-clojure), but have also worked on grammars for Fennel and Janet.

After discovering that testing the grammar on real-world code can turn up all sorts of things I missed or got wrong, I started writing [property-based tests](https://github.com/sogaiu/prop-test-ts-clj) for it using Python's hypothesis testing library and [py-tree-sitter](https://github.com/tree-sitter/py-tree-sitter).  This has also turned up some unexpected things which has helped improve the grammar.
